src/ImageSharp/Quantizers/WuQuantizer{TPixel}.cs

```diff
@@ -197,19 +196,21 @@ protected override void InitialQuantizePixel(TPixel pixel)
             var rgba = default(Rgba32);
             pixel.ToRgba32(ref rgba);
 
-            int r = rgba.R >> 2; // 8 - IndexBits
-            int g = rgba.G >> 2;
-            int b = rgba.B >> 2;
-            int a = rgba.A >> 5; // 8 - IndexAlphaBits
+            int r = rgba.R >> (8 - IndexBits);
```

Comment:
This is where the bug was, I was adding the shifted component instead of the originals, the rest is optimization code.